daemon/cmd/daemon_main.go

```diff
@@ -216,9 +216,12 @@ func init() {
 	})
 
 	// Env bindings
-	flags.Int(option.AgentHealthPort, defaults.AgentHealthPort, "TCP port for agent health status API")
+	flags.Int(option.AgentHealthPort, defaults.AgentHealthPort, "TCP port for agent health status API inside agent pod")
```

Comment:
nit: I think this is a bit misleading. Cilium isn't required to run inside a pod, even though the vast majority of users will do so. Additionally, Cilium usually runs in host network so that port is usually exposed outside the pod by default. I'd skip this text update change.

Reply:
Yeah, I wasn't sure about this, but the text for each of these flags needs to reflect the difference, and I'm not sure what the difference is. Will think about it a bit more...